app/signer/bls_test.go

```diff
@@ -366,11 +374,12 @@ func TestLoadBlsSignerIfExists(t *testing.T) {
 
 		nonExistentKeyFile := filepath.Join(tempDir, "non-existent-key.json")
 
-		blsSigner := LoadBlsSignerIfExists(tempDir, "", nonExistentKeyFile)
+		blsSigner, err := LoadBlsSignerIfExists(tempDir, "", nonExistentKeyFile)
+		assert.NoError(t, err)
 		assert.Nil(t, blsSigner, "Should return nil when key file doesn't exist")
 	})
 
-	t.Run("return nil when neither env var nor password file exists", func(t *testing.T) {
+	t.Run("return error when neither env var nor password file exists", func(t *testing.T) {
 		originalValue := os.Getenv(BlsPasswordEnvVar)
```

Comment:
to determine env variable we should use `LookupEnv` which can differentiate between cases:
- no passphrase provided by env variable
- empty passphrase (`""`) provided by env variable

Reply:
What I mean, is that our function https://github.com/babylonlabs-io/babylon/blob/main/app/signer/bls.go#L150, should use `LookupEnv`, as if somebody creates keyring with empty passphrase, he can later provided it by env variable.  (and with `GetEnv` we would not recognize it)